tests/pytest/core/test_models.py

```diff
@@ -247,6 +247,16 @@ def test_EligibilityType_enrollment_index_template(model_EligibilityType):
     assert model_EligibilityType.enrollment_index_template == "test/enrollment.html"
 
 
+@pytest.mark.django_db
+def test_EligibilityType_enrollment_success_template(model_EligibilityType):
```

Comment:
I think this test can be updated slightly to assert what we changed here -- the `EligibilityType.enrollment_success_template` should have a _default value_ of `enrollment/success.html`.

As written, this test is asserting that the fixture `model_EligibilityType` was configured with that value (in the conftest).

Instead, we should test that creating a brand new `EligibilityType` should get the default value.

Reply:
In other words, this test should not _inject the fixture_ but instead create the object directly in the test (with no value given for the template) and assert the default.

Reply:
Thanks! This makes sense, I updated the test to create the object and then assert the default value. I had a choice to use 

- `new_eligibility_type = EligibilityType.objects.create()` or 
- `new_eligibility_type = EligibilityType()` 

to create the object, but it seemed that `new_eligibility_type = EligibilityType.objects.create()` was more appropriate for this test since it [creates and saves the object](https://docs.djangoproject.com/en/5.0/topics/db/queries/#creating-objects) even though either command seemed to work for the test.

Reply:
Oh and I guess this may be out of scope for this PR (not because it's a huge change but just better to leave it out of this PR), but I could also update `test_EligibilityType_enrollment_index_template()` since it seems that it also tests the fixture when it should test the creation of a new object.

Reply:
I think that's a good idea for a follow-up to align these tests :+1: 

There is a _slight_ difference here (and it isn't obvious when looking at just the test) in that the fixture didn't set the `enrollment_index_template` so this test is in fact asserting the default. But I agree it would be better to do it in the same way -- namely, creating the model directly in the test and asserting the default.

Reply:
Great, that sounds good! But yeah, I see the difference, the fixture didn't set `enrollment_index_template` so it is testing the behavior we want to test.

Reply:
Noted this in a follow-up #2108